general/server/media/movies.md

```diff
@@ -50,3 +49,14 @@ If labels are not added to the end of filenames, as shown above, each file will
 
 > [!Note]
 > To group media manually, long-click or right-click media to highlight then select additional media to merge. Use the new bar that appears to 'Group Versions'.
+
+### Order of Versions
+
+Movie versions are presented in an alphabetically sorted list. An exception applies to resolution names, which are sorted in descending order from largest to smalles resolution. A version name qualifies as a resolution name when ending with either a `p` or an `i`.
```

Comment:
```suggestion
Movie versions are presented in an alphabetically sorted list. An exception applies to resolution names, which are sorted in descending order from largest to smallest resolution. A version name qualifies as a resolution name when ending with either a `p` or an `i`.
```
Preferably, one would even write "highest to lowest resolution".

Reply:
Thanks for the suggestion. I agree with your change request.